lib/shopify_transporter/pipeline/magento/product/top_level_attributes.rb

```diff
@@ -50,6 +50,15 @@ def append_images(input)
               images.sort_by { |image| image['position'] }
             end
 
+            def append_images(input)
```

Comment:
Is there a way to do this without using mutation? What motivated the use of mutation here?

Reply:
At the _very_ least, I'd want it to be `append_images_to_output!` or something -- a more descriptive name that describes what it is doing and indicates its mutative nature.